lib/cube_browser/__init__.py

```diff
@@ -387,6 +387,11 @@ def __call__(self, **coord_values):
 
         """
         cube = self._get_slice(coord_values)
+        # guess bounds, if required.
+        for cname in self.coords:
+            coord = cube.coord(cname)
+            if len(coord.points) > 1 and not coord.has_bounds():
+                coord.guess_bounds()
```

Comment:
@marqh Really? This has the side-effect of changing the cube.

I'd argue that the user should prep the cube beforehand i.e. ensure that it has bounds, rather that convenience magic occurring with cube_browser ... thoughts?